probit/src/main/scala/com/circlegain/exmods/probit/internal/models/Trade.scala

```diff
@@ -0,0 +1,49 @@
+package com.circlegain.exmods.probit.internal.models
+
+import java.time.Instant
+
+import cats.Monad
+import cats.effect.Clock
+import com.circlegain.exmods.core.make.Make
+import com.circlegain.exmods.core.models.domain
+import com.circlegain.exmods.core.models.domain._
+import com.circlegain.exmods.core.models.domain.errors.MalformedResponse
+import io.circe.Decoder
+import io.circe.generic.extras.semiauto.deriveConfiguredDecoder
+import tofu.Raise
+import tofu.syntax.monadic._
+
+final case class Trade(
+  id: String,
+  orderId: String,
+  time: Instant,
+  side: OrderSide,
+  quantity: BigDecimal,
+  price: BigDecimal,
+  feeAmount: BigDecimal,
+  feeCurrencyId: String
+)
+
+object Trade {
+  implicit val decoder: Decoder[Trade] = deriveConfiguredDecoder
+
+  implicit def makeDomain[F[_]: Monad: Clock: Raise[*[_], MalformedResponse]](implicit
+    O: Make[OrderSide, domain.OrderType, F]
```

Comment:
Lets use more meaningful names for custom modules such as Make. 
```suggestion
    makeOrderType: Make[OrderSide, domain.OrderType, F]
```